envs/example/defaults.yml

```diff
@@ -21,6 +21,7 @@ secrets:
   horizon_secret_key:           asdf
   glance_sync:      ADQ64XUQLUWH75M634RVBLP55RKPGGOWG
   monitor_password: asdf
+  telemetry_password: asdf
```

Comment:
what is the difference between this and the ceilometer password ( secrets.service_password ) ?